### PR TITLE
Fix steering backlog link.

### PR DESCRIPTION
**Pull request by mrbobbytables**
🤦‍♂ Used the old link.

### CODE REVIEW

File: content/en/blog/_posts/2019-10-03-2019-Steering-Committee-Election-Results.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Steering Committee Meetings:
 [@pwittrock]: https://github.com/pwittrock
 
 [CIVS]: https://civs.cs.cornell.edu/
-[backlog items]: https://git.k8s.io/steering/backlog.md
+[backlog items]: hhttps://github.com/kubernetes/steering/projects/1
```
Comment (kbhawkey) [inline content/en/blog/_posts/2019-10-03-2019-Steering-Committee-Election-Results.md]: @mrbobbytables , fix the https://?

Comment (mrbobbytables) [reply content/en/blog/_posts/2019-10-03-2019-Steering-Committee-Election-Results.md]: gah, double 🤦‍♂ just fixed.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-16683--kubernetes-io-master-staging.netlify.com/blog/

**Comment by mrbobbytables** [comment]
Much appreciated @kbhawkey. Thanks!

**Pull request merged**